src/PostList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type Props = {
+  posts: Post[];
+};
+
+const PostsList: React.FC<Props> = ({ posts }) => (
+  <ul>
+    {posts.map((post) => (
+      <li>
```

Comment:
Every time we map over itens in React and render then we need to set up the 'key' property. 

https://react.dev/learn/rendering-lists